website/source/docs/providers/aws/r/cloudfront_distribution.html.markdown

```diff
@@ -0,0 +1,197 @@
+---
+layout: "aws"
+page_title: "AWS: cloudfront_distribution"
+sidebar_current: "docs-aws-resource-cloudfront-distribution"
+description: |-
+  Provides a CloudFront web distribution resource.
+---
+
+# aws\_cloudfront\_distribution
+
+Creates an Amazon CloudFront web distribution.
+
+For information about CloudFront distributions, see the
+[Amazon CloudFront Developer Guide][1]. For specific information about creating
+CloudFront web distributions, see the [POST Distribution][2] page in the Amazon
+CloudFront API Reference.
+
+~> **NOTE:** CloudFront distributions take about 15 minutes to a deployed state
+after creation or modification. During this time, deletes to resources will be
+blocked. If you need to delete a distribution that is enabled and you do not
+want to wait, you need to use the `retain_on_delete` flag.
+
+## Example Usage
+
+The following example below creates a CloudFront distribution with an S3 origin.
+
+```
+resource "aws_cloudfront_distribution" "s3_distribution" {
+	origin {
+		domain_name = "mybucket.s3.amazonaws.com"
+		origin_id = "myS3Origin"
+		s3_origin_config {
+      origin_access_identity = "origin-access-identity/cloudfront/ABCDEFG1234567"
+    }
+	}
+	enabled = true
+	comment = "Some comment"
+	default_root_object = "index.html"
+	logging_config {
+		include_cookies = false
+		bucket = "mylogs.s3.amazonaws.com"
+		prefix = "myprefix"
+	}
+	aliases = [ "mysite.example.com", "yoursite.example.com" ]
+	default_cache_behavior {
```

Comment:
In the documentation below it says that `path_pattern` is required, yet in this example it is absent. One or the other should be updated.